gammapy/image/tests/test_mask.py

```diff
@@ -16,7 +16,11 @@ def test_random_creation():
     assert excluded[0].size != 0
 
 
-@pytest.mark.xfail
 def test_distance_image():
-    pass
-    # TODO
+    mask = ExclusionMask.empty(nxpix=300, nypix=200)
```

Comment:
Please make the test image smaller, so that the result can easily be printed and looked at.

It would also be good to have three input test images for this:
* Everything excluded
* Nothing excluded
* Some pixels excluded, maybe an edge and center pixel or block of few pixels.